fixtures/build-all.py

```diff
@@ -0,0 +1,9 @@
+import TUFTestFixtureSimple
+
+from unittest import mock
+
+@mock.patch('time.time', mock.MagicMock(return_value=1577836800))
+def build_all():
+    TUFTestFixtureSimple.build()
+
+build_all()
```

Comment:
Do we actually need this new file? 
Can't we just update `generate_fixtures.py`

1. Delete the line that calls TUFTestFixtureSimple()
2. replace it with TUFTestFixtureSimple.build()

Once we replace all the fixture generation  `generate_fixtures.py` would only have TUFTestFixture*.build() calls and all class could be removed.
Otherwise right now the readme is not accurate as far as generating fixtures and it would be a 2 step process.

Reply:
@phenaproxima  to save time pushed a changed removing `fixtures/build-all.py` for now. I am calling `fixtures.TUFTestFixtureSimple.build()` from  `generate_fixtures.py`. This way all the fixtures are generated from 1 file as documented in the readme.

Once we have converted all fixtures we can either delete `generate_fixtures.py` and add back `fixtures/build-all.py` or just leave `generate_fixtures.py` with all the new builders.

Let me if this works for you.